bin/companion.js

```diff
@@ -0,0 +1,2 @@
+require('dotenv').config({ path: '.env.local' })
```

Comment:
That makes me think we should probably go back to using `.env` instead of `.env.local`. I thought `.env.local` was standard and would be picked up by dotenv by default, but it looks like it's not the case...

Reply:
Ah, I thought we initially had a `.env` file too in addition to `.env.local`, although I cannot find it anywhere. Then I thought .env.local was local (dev) overrides for `.env`. That is how I'm used to it working in [create-react-app](https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used).
But unless we want to commit a `.env` with default values, then I agree we can instead just use `.env`, because it doesn't seem like dotenv supports .env.local

Reply:
Yes, my original idea was to have a `.env` file that would contain the default values, and have a `.env.local` file that would override it. I gave up on that and went back on having only one file because:

1. dotenv docs explicitly recommend against that: https://github.com/motdotla/dotenv#should-i-have-multiple-env-files.
2. It made overriding config for the e2e CI tricky.

Let me open a PR to address that.

Reply:
Ah. Yes `dotenv` also recommends against commiting .env files or env variables. Same with the 12 factor app. But then dotenv is more geared towards backend node.js apps where .env files contain actual secrets. [create-react-app on the other hand](https://github.com/facebook/create-react-app/issues/2403) does recommend commiting .env files. But then on the flipside because frontend apps will usually expose all env variables in the bundled code anyways, it doesn't make sense to try to keep them secret. Our .env is a mix between backend and frontend, so then who knows what's right 🤷‍♂️

But I think keeping .env out of git is the only safe common denominator